compiler/src/dmd/expression.d

```diff
@@ -1358,7 +1356,21 @@ extern (C++) abstract class Expression : ASTNode
 
     final int isConst()
     {
-        return .isConst(this);
+        //printf("Expression::isConst(): %s\n", e.toChars());
+        switch (op)
+        {
+        case EXP.int64:
+        case EXP.float64:
+        case EXP.complex80:
+            return 1;
+        case EXP.null_:
+            return 0;
+        case EXP.symbolOffset:
+            return 2;
```

Comment:
is `2` vs `1` still used nowadays ?

Reply:
Nowadays, probably not, but when this code was written....

Reply:
At least for clarity the return should be an enum member. I understand that this is out of the scope this PR tho.